### ISSUE TITLE
Docker image for java 17 doesn't work for mac M1

### ISSUE THREAD

**Issue by singhanee** [personio/backend-coding-challenge#9]
This has been introduced in the latest commit https://github.com/personio/backend-coding-challenge/pull/8/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R1

```
 => ERROR [internal] load metadata for docker.io/library/eclipse-temurin:17-jre-alpine                       1.3s
------
 > [internal] load metadata for docker.io/library/eclipse-temurin:17-jre-alpine:
------
failed to solve: eclipse-temurin:17-jre-alpine: no match for platform in manifest sha256:8a139831f95086a107f510f239cb311d4196b94f0f15192a22f8f7a43f343d02: not found

No matching manifest for linux/arm64/v8 in the manifest list entries
```

**Comment by joschi** [comment]
@singhanee Thank you very much for reporting this!

While it is possible to run `amd64` images on systems with Apple Silicon (M1, M2) using Rosetta 2, it shouldn't be necessary for this project since alternatives are available.

This being said, here are the instructions to install Rosetta 2 on macOS for use with Docker Desktop:
https://docs.docker.com/desktop/install/mac-install/#mac-with-apple-silicon

**Comment by joschi** [comment]
Fixed in b55749d6540944b6c01eae6e815be35dddf778f9 by using `eclipse-temurin:17-jre-jammy`.

```
# docker inspect eclipse-temurin:17-jre-jammy -f '{{.Architecture}}'
arm64
```
